docs/quickstarts/fastify.mdx

```diff
@@ -27,171 +64,166 @@ After following this guide, you should have a working Fastify app with public an
 
-  ```ts {{ filename: 'index.ts' }}
-  import * as dotenv from 'dotenv'
+  [`clerkClient`](/docs/references/backend/overview) is an instance of Clerk's JavaScript Backend SDK, which provides access to Clerk's Backend API resources and low-level  utilities for handling authentication in JavaScript environments. With `clerkClient`, you can manage users, sessions, and metadata by interacting directly with Clerk's API.
 
-  dotenv.config()
+  The following example uses `getAuth()` to retrieve the `userId`, which is used to protect the route and is passed to `clerkClient.users.getUser()` to retrieve the current user's `User` object.
 
+  ```ts {{ filename: 'index.ts', mark: [3, [9, 17]] }}
+  import 'dotenv/config'
   import Fastify from 'fastify'
   import { clerkClient, clerkPlugin, getAuth } from '@clerk/fastify'
 
   const fastify = Fastify({ logger: true })
 
   fastify.register(clerkPlugin)
 
-  fastify.get('/', async (req, reply) => {
-    /**
-     * Access the auth state for this request.
-     * In this example, the userId loads the whole User object
-     * from the Clerk servers
-     */
-    const { userId } = getAuth(req)
+  fastify.get('/', async (request, reply) => {
+    const { userId } = getAuth(request)
     const user = userId ? await clerkClient.users.getUser(userId) : null
-    return { user }
+
+    return reply.send({
+      message: 'Authentication state retrieved successfully.',
+      user,
+    })
   })
 
   const start = async () => {
     try {
-      await fastify.listen({ port: 3000 })
-    } catch (err) {
-      fastify.log.error(err)
+      await fastify.listen({ port: 8080 })
+    } catch (error) {
+      fastify.log.error(error)
       process.exit(1)
     }
   }
 
   start()
   ```
 
-  ### Require authentication for a route
-
-  To protect a route using Clerk, you can use `getAuth` to check if the user is authenticated. If the user is not authenticated, you can return a 403 error.
+  To protect a route using Clerk, use the `getAuth()` method to check if the user is authenticated. If not, it returns an error response. If authenticated, it retrieves the user information using `clerkClient`.
 
   ```ts {{ filename: 'index.ts' }}
   fastify.get('/protected', async (request, reply) => {
-    const { userId } = getAuth(request)
-    if (!userId) {
-      return reply.code(403).send()
-    }
+    try {
+      const { userId } = getAuth(request)
 
-    const user = await clerkClient.users.getUser(userId)
-    return { user }
+      if (!userId) {
+        return reply.code(403).send({ error: 'Unauthorized request.' })
+      }
+
+      const user = await clerkClient.users.getUser(userId)
+
+      return reply.send({
+        message: 'Authentication state retrieved successfully.',
+        user,
+      })
+    } catch (error) {
+      fastify.log.error(error)
+      return reply.code(500).send({ error: 'Failed to retrieve user information.' })
+    }
   })
   ```
 </Steps>
```

Comment:
So what I was saying in the comment before about merging the code examples, as shown [here](https://github.com/clerk/clerk-docs/pull/1201/files#diff-b704145027aa89795fd84c206c02c3f07082daaa6be1549c2f531aede6330774R122)

all you have you do is throw
`if (!userId) {
        return reply.code(403).send({ error: 'Unauthorized request.' })
      }`

into the code example above it.

having both of these examples is extremely redundant. they're doing the exact same thing, except the first doesn't have an auth check and the second does.